<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pocket ImAT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MyView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1900</width>
        <height>1280</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuLocale">
     <property name="title">
      <string>Locale</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionEnglish_US"/>
     <addaction name="actionPortuguese_BR"/>
    </widget>
    <addaction name="actionUse_line"/>
    <addaction name="actionSet_Scale"/>
    <addaction name="menuLocale"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionUse_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Circled Line</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionSet_Scale">
   <property name="text">
    <string>Set Scale</string>
   </property>
  </action>
  <action name="actionEnglish_US">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English - US</string>
   </property>
  </action>
  <action name="actionPortuguese_BR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Portuguese - BR</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header>myview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
